packages/rocketchat-message-snippet/server/publications/snippetedMessage.js

```diff
@@ -0,0 +1,33 @@
+Meteor.publish('snippetedMessage', function(_id) {
+	if (typeof this.userId === 'undefined' || this.userId === null) {
+		return this.ready();
+	}
+
+	let publication = this;
+
+	let user = RocketChat.models.Users.findOneById(this.userId);
+
+	if (typeof user === 'undefined' || user === null) {
+		return this.ready();
+	}
+
+	let cursor = RocketChat.models.Messages.find(
+		{ '_id': _id }
+	).observeChanges({
+		added: function(_id, record) {
+			publication.added('rocketchat_message', _id, record);
```

Comment:
We should not use the real collection name here, can you replace by `rocketchat_snippeted_message`